testing/src/scenario/state.py

```diff
@@ -1692,18 +1688,19 @@ class DeferredEvent:
     # them because they need a Handle.
     @property
     def name(self):
+        """A comparable name for the event."""
         return self.handle_path.split("/")[-1].split("[")[0]
 
 
 class _EventType(str, Enum):
-    framework = "framework"
```

Comment:
Huh, I guess inheriting from `str` and `Enum` is a way to get `StrEnum` like behaviour for older pythons

Reply:
Yeah, that's what people usually do/did from my experience. It's not perfect, but I guess close enough that people found it ok.